code/obj/item/building_materials.dm

```diff
@@ -1321,7 +1321,7 @@ ABSTRACT_TYPE(/datum/sheet_crafting_recipe/zwood)
 		fl_tiles
 			recipe_id = "fl_tiles"
 			craftedType = /obj/item/tile
-			name = "Floor Tile"
+			name = "Floor Tiles"
```

Comment:
This is the display name btw I swapped the actionbar outputting this name into chat to do the obj name instead because like, 
![dreamseeker_AW9LxiWFep](https://github.com/goonstation/goonstation/assets/27376947/bc8c99d7-8e2a-43d6-8cf6-407017f5c9a2)
This is bad
SERIOUSLY HOW HAS NO-ONE ELSE SEEN THIS???